ui/shared/components/panel/family.jsx

```diff
@@ -5,23 +5,27 @@ import { Grid, Popup, Icon } from 'semantic-ui-react'
 import { Link } from 'react-router-dom'
 import styled from 'styled-components'
 
-import { updateFamily } from 'redux/rootReducer'
-import { getProjectsByGuid, getFirstSampleByFamily } from 'redux/selectors'
+import { updateFamily, loadStaffOptions } from '../../../redux/rootReducer'
+import { getProjectsByGuid, getFirstSampleByFamily, getUserOptionsIsLoading, getUsersByUsername } from '../../../redux/selectors'
```

Comment:
For imports from outside the same top level folder it is actually better style to use the absolute import (i.e. `'redux/selectors'` instead of the relative import (i.e. `'../../../redux/selectors`')